src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,12 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
-
+    
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.